### PR TITLE
feat(options): Display biometrics option without setting password beforehand

### PR DESCRIPTION
**Pull request by ak-ledger**
### 📝 Description

Add the option greyed out if no password setted. Modified privacy settings logics. Add analytics to password and biometrics options.
### ❓ Context

- **Impacted projects**: `LLM`
- **Linked resource(s)**: [LIVE-2398] 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 📸 Demo

Simulator Screenshot - iPhone 14 - 2023-10-11 at 17 43 09

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-2398]: https://ledgerhq.atlassian.net/browse/LIVE-2398?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mcayuelas-ledger** [comment]
Screenshot 2023-10-13 at 09 25 59
Followint this screenshot, you need to setup a password and then activate FaceID right?

File: apps/ledger-live-mobile/src/screens/Settings/General/BiometricsRow.tsx [hunk 1]
```diff
@@ -26,8 +32,19 @@ export default function BiometricsRow({ iconLeft }: Props) {
   const biometricsType =
     t(`auth.enableBiometrics.${privacy?.biometricsType?.toLowerCase()}`) ?? privacy?.biometricsType;
 
+  /** CALLBACKS */
+
   const onValueChange = useCallback(
     async (biometricsEnabled: boolean) => {
+      console.error(validationPending);
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/screens/Settings/General/BiometricsRow.tsx:39]: clear console 🔥

File: apps/ledger-live-mobile/src/components/SettingsRow.tsx [hunk 2]
```diff
@@ -5,25 +5,33 @@ import { ChevronRightMedium, InfoMedium } from "@ledgerhq/native-ui/assets/icons
 import styled from "styled-components/native";
 import Touchable from "./Touchable";
 
-const StyledTouchableRow = styled(Touchable)<{
+const StyledTouchableRowContainer = styled(Flex)<{
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/components/SettingsRow.tsx:8]: You can have called it juste like that as it's not touchable 
```suggestion
const StyledRowContainer = styled(Flex)<{
```

Just a minor comment can be ignored !

**Review by mcayuelas-ledger — APPROVED** [review]
LGTM ! 
Could be goo ti have a screenrecord when faceId is enabled !

File: apps/ledger-live-mobile/src/screens/Settings/General/AuthSecurityToggle.tsx [hunk 3]
```diff
@@ -7,17 +7,32 @@ import { NavigatorName } from "../../../const";
 import { privacySelector } from "../../../reducers/settings";
 import SettingsRow from "../../../components/SettingsRow";
 import BiometricsRow from "./BiometricsRow";
+import { ScreenName } from "../../../const";
+import { track } from "../../../analytics";
 
+/** COMPONENT */
```
Comment (sshmaxime) [inline apps/ledger-live-mobile/src/screens/Settings/General/AuthSecurityToggle.tsx:13]: I think we should avoid doing this

**Review by sshmaxime — CHANGES_REQUESTED** [review]

File: apps/ledger-live-mobile/src/screens/Settings/General/AuthSecurityToggle.tsx (hunk 3 above)
Comment (valpinkman) [reply apps/ledger-live-mobile/src/screens/Settings/General/AuthSecurityToggle.tsx:13]: agreed with  @sshmaxime here

File: apps/ledger-live-mobile/src/screens/Settings/General/AuthSecurityToggle.tsx [hunk 4]
```diff
@@ -7,17 +7,27 @@ import { NavigatorName } from "../../../const";
...
   const { t } = useTranslation();
 
   const privacy = useSelector(privacySelector);
   const { navigate } = useNavigation();
 
-  const onValueChange = (authSecurityEnabled: boolean): void =>
+  const onValueChange = (authSecurityEnabled: boolean): void => {
+    // Add onClick Analytics
+    track("toggle_clicked", {
+      toggle: "Password Lock", // Moyen de chopper la key non traduite ?
```
Comment (sshmaxime) [inline apps/ledger-live-mobile/src/screens/Settings/General/AuthSecurityToggle.tsx:22]: No, AFAIK stuff like that are in English. You can also remove the `// Add onClick Analytics` it's already pretty straightforward IMO.

**Review by sshmaxime — CHANGES_REQUESTED** [review]

**Review by sshmaxime — APPROVED** [review]

**Pull request merged**